src/components/map-page-components/mobile-map-page/MobileMapPage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Layout } from 'antd';
+import MapView from '../map-view/MapView';
+import MobileFooter from '../../../mobile-footer/MobileFooter';
+
+const { Content, Footer } = Layout;
+
+const MainContent = styled.div`
+  height: 100%;
+  margin-bottom: 15px;
+`;
+
+const MobileMapPage: React.FC = ({ children }) => {
+  return (
+    <>
+      <MainContent>
+        <Layout style={{ height: 'calc(100vh - 83px)' }}>
```

Comment:
This is a powerful string to have in an inline style. I believe this height calculation is used many places, can it be abstracted to a constant somewhere?

Reply:
We could put the constant somewhere - maybe in `window-dimensions.tsx`? Alternatively, I think it's used for the map layouts only so we could make a `MapLayout` component with this inline styling.

Reply:
If it's only map pages, then I think a MapLayout component would be good. Otherwise use window dimensions.